modules/conversion-center/assets/js/editor/hooks/ui/editor/documents/open/add-contact-buttons-tab.js

```diff
@@ -17,7 +17,7 @@ export class ContactPageAddLibraryTab extends $e.modules.hookUI.After {
 			title: __( 'Floating Buttons', 'elementor' ),
 			filter: {
 				source: 'remote',
-				type: 'contact_button',
+				type: 'floating_button',
```

Comment:
@nicoladj77 Maybe this should be in it's own PR. I think we're gonna have some back and forth on this PR.